### DISCUSSION TITLE
Allow users to publish drafts directly from live preview

### DISCUSSION THREAD

**Discussion by jsunsawyer** [craftcms/cms#9010] [Ideas] [answered]
Currently, to make a quick edit in live preview, the user is required to:

1. Click Preview
2. Make edit
3. Close Preview
4. Click Keep It
5. Click Publish Draft

This should simply be:

1. Click Preview
2. Make edit
3. Click Save (or Publish – whatever language works best here)

**Comment by bencresty** [comment]
Had requested the similar  thing, but Brandon Kelly let me know you already had this one.

So definitely +1 for me too!

**Reply by jsunsawyer** [reply]
@bencresty I liked your suggestion of keeping live preview open on "save and continue editing" in your request. It seems like sensible functionality for users who prefer live preview.

**Reply by bencresty** [reply]
@jsunsawyer Thanks. It seems like our requests are pretty similar though. Love the preview interface, a save button there would really compliment it! Have a nice day

**Comment by brandonkelly** [comment, marked as answer]
Live Preview is getting a dedicated “Save” button in Craft 5.2 🎉 (#15112)

**Reply by brandonkelly** [reply]
Craft 5.2.0-beta.1 is out with those changes.
